src/components/MultivariateLegend/ExtrusionStepsLegend.tsx

```diff
@@ -0,0 +1,26 @@
+import { PrismHigh, PrismLow, PrismMed } from '@konturio/default-icons';
+import { MultivariateLegendStep } from './MultivariateLegendStep';
+
+export default function ExtrusionStepsLegend({
+  lowMCDAScoreLayersDirections,
+  highMCDAScoreLayersDirections,
+}: {
+  lowMCDAScoreLayersDirections: string[];
+  highMCDAScoreLayersDirections: string[];
+}) {
+  return (
+    <>
+      <MultivariateLegendStep
+        textLines={highMCDAScoreLayersDirections}
+        icon={<PrismLow />}
+        lineKey="extrusion-low"
```

Comment:
PrismLow icon is used for highMCDAScoreLayersDirections
PrismHigh icon is used for lowMCDAScoreLayersDirections

Please ensure it is by design.